JustBrewIt/Pages/Brewery.cshtml.cs

```diff
@@ -18,9 +18,16 @@ public void OnGet()
 
             using (var webClient = new WebClient())
             {
-                string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries?by_city=cincinnati&brewery_type=regional");
-                Welcome[] welcome = Welcome.FromJson(jsonString);
-                ViewData["MyBreweryAPI"] = welcome;
+                try
+                {
+                    string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries?by_city=cincinnati&brewery_type=regional");
+                    Welcome[] welcome = Welcome.FromJson(jsonString);
+                    ViewData["MyBreweryAPI"] = welcome;
+                }
+                catch(Exception)
+                {
+                    Console.WriteLine("Something went wrong!");
```

Comment:
Exception handling is always a good idea, but take it all the way.  This is catching an exception, throwing it away, and then assuming the program worked perfectly (which it did not).  The user will have no idea.
1) Inform the user with an appropriate message.  Don't give away too many details, but indicate what is unavailable.
2) Use proper logging instead of Console.WriteLine().
- Loggins has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().